test/studies/shootout/submitted/chameneosredux-submitted.graph

```diff
@@ -1,5 +1,5 @@
-perfkeys: real, real, real
-graphkeys: release, submitted, release-fast
-files: chameneosredux.dat, chameneosredux-submitted.dat, chameneosredux-fast.dat
+perfkeys: real, real, real, real
+graphkeys: release, submitted, release-fast, submitted (2)
```

Comment:
Why not "submitted-fast"?

Reply:
nvm, I see this is the same thing we do in fasta-submitted.graph, so this is fine by me

Reply:
This is what we've done in other cases (e.g., fasta) to match the numbering used on the CLBG website since it doesn't use descriptive labels like "fast".